svo (inker).xml

```diff
@@ -93,11 +93,9 @@
                 <colorTriggerFgColor>#000000</colorTriggerFgColor>
                 <colorTriggerBgColor>#000000</colorTriggerBgColor>
                 <regexCodeList>
-                    <string>As the</string>
```

Comment:
You should leave this trigger as a `multiline` trigger. It is an optimisation because RegEx triggers are relatively expensive performance wise. Begin of line triggers are the cheapest kind of trigger and as such act as a filter in this case.